R/z_animint.R

```diff
@@ -285,6 +285,10 @@ animint2dir <- function(plot.list, out.dir = NULL,
   if(!is.null(plot.list$out.dir)){
     plot.list$out.dir <- NULL
   }
+  if(is.character(plot.list[["source"]])){
+    meta$source <- plot.list[["source"]]
+    plot.list$source <- NULL
```

Comment:
this allows the user to specify `source` as one of the plot options, it should be a URL where we can find the source code for this data viz.